sample-notificator-app/src/main/resources/notification-template/email-update-template.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>email-update-template</title>
+</head>
+<body>
+<div id="email-body">
+  <div id="person">Dear Max Mustermann,</div>
+  <br>
+  <div id="content">this is an automated email to inform you about recent updates on your subscribed project
+    <div id="project-information">My awesome project (ABCDE):</div>
+    <br>
+    <div id="sample-status-enumeration">
+      <div id="sample-status-failed-qc"><strong>X samples failed QC</strong>
+        <span id ="failed-qc-contact">(get in contact: <a href="mailto:support@qbic.zendesk.com">support@qbic.zendesk.com</a>)</span>
+      </div>
+      <div id="sample-status-available-data">X samples data available</div>
+    </div>
+    <br>
+    </div>
+    <div id="subscription-information">Click <a href="https://portal.qbic.uni-tuebingen.de/portal/">here</a>
```

Comment:
I the `here` intended to lead to the portal?

Reply:
Currently we have no dedicated readme or content for detailing the subscription information. @jenniferboedker suggested that for now a linkage to the portal would suffice